src/index-metadbs/README.md

```diff
@@ -0,0 +1,41 @@
+# Static Index Meta-Databases
+
+## Goal
+
+This folder hosts static versions of Index Meta-Databases for those providers that only have one main sub-database (or very few sub-databases) and do not wish to maintain an Index Meta-Database themselves (Index Meta-Databases are described in the [OPTIMADE specifications](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst)).
```

Comment:
Do you want to also use the header tag in this link, or?

Reply:
I removed it on purpose as I wanted to point to v1 once is released, and I feel we'll forget to fix this file (while we will remember the README). So I preferred to keep a generic link.
I can put it back if you want. But we'll need to remember to adapt it once the specs are out.

Reply:
I see now that in the rest you added them. I'll then do the same.

Reply:
Yeah, I'll create a preemtive issue stating these links should be updated come the release of v1.0.